agent_paths_id[] path_array
real_robot_controller/agent_job[] agent_job
int16 max_job_size
